src/plugins/intel_npu/tests/functional/internal/overload/driver_caching.hpp

```diff
@@ -0,0 +1,290 @@
+
+    std::string secondCompilationDriverLog = ::intel_npu::zeroUtils::getLatestBuildError(graph_ddi_table_ext);
+    std::printf("==[1.3][EmptyConfig] driver log content : #%s#\n", secondCompilationDriverLog.c_str());
+    EXPECT_TRUE(containsCacheStatus(secondCompilationDriverLog, "cache_status_t::found"));
+
+    std::printf("==[1.4]testsuit time (1): %f, (2): %f\n", durationFirst.count(), durationSecond.count());
+    EXPECT_LT(durationSecond, durationFirst)
+        << "The duration of the second compilation should be less than the first due to UMD Caching.";
+
+    checkCacheDirectory();
+}
+
+TEST_P(CompileAndDriverCaching, CompilationCacheWithOVCacheConfig) {
+    checkCacheDirectory();
+    ze_graph_dditable_ext_decorator& graph_ddi_table_ext = initStruct->getGraphDdiTable();
+    
+    //Check the initial state if this testp is called separately
+    std::string driverLogInitContent = ::intel_npu::zeroUtils::getLatestBuildError(graph_ddi_table_ext);
+    std::printf("==[2.1][OVCacheConfig] driver log content : #%s#\n", driverLogInitContent.c_str());
+
+    //ZE DynamicCaching cache_status_t or empty
+    // if (!driverLogInitContent.empty() && !containsCacheStatus(driverLogInitContent, "ZE DynamicCaching cache_status_t")) {
+    //         FAIL() << "Due TO NOW UMD CACHE LOG ";
+    // }
+
+    configuration[ov::cache_dir.name()] = "./testCacheDir";
+    m_cachedir = configuration[ov::cache_dir.name()].as<std::string>();
+    ov::CompiledModel execNet;
+    function = getConstantGraph();
+
+    //first compilation will long and will generate the model cache.
+    auto startFirst = std::chrono::high_resolution_clock::now(); 
+    OV_ASSERT_NO_THROW(execNet = core->compile_model(function, target_device, configuration));
+    auto endFirst = std::chrono::high_resolution_clock::now();
+    std::chrono::duration<double> durationFirst = endFirst - startFirst;
+
+    std::string firstCompilationDriverLog = ::intel_npu::zeroUtils::getLatestBuildError(graph_ddi_table_ext);
+    std::printf("==[2.2][OVCacheConfig] driver log content : #%s#\n", firstCompilationDriverLog.c_str());
+    EXPECT_TRUE(containsCacheStatus(firstCompilationDriverLog, driverLogInitContent));
+    
+    // EXPECT_TRUE(!containsCacheStatus(firstCompilationDriverLog, "cache_status_t::stored") &&   
+    //         !containsCacheStatus(firstCompilationDriverLog, "cache_status_t::found"));
+
+    checkCacheDirectory();
+    //second  compilation
+    auto startSecond = std::chrono::high_resolution_clock::now(); 
+    OV_ASSERT_NO_THROW(execNet = core->compile_model(function, target_device, configuration));
+    auto endSecond = std::chrono::high_resolution_clock::now();
+    std::chrono::duration<double> durationSecond = endSecond - startSecond;
+
+    std::string secondCompilationDriverLog = ::intel_npu::zeroUtils::getLatestBuildError(graph_ddi_table_ext);
+    std::printf("==[2.3][OVCacheConfig] driver log content : #%s#\n", secondCompilationDriverLog.c_str());
```

Comment:
Now, this code can work On windows.
On linux, replace by `EXPECT_TRUE(!containsCacheStatus(secondCompilationDriverLog, "cache_status_t::stored") &&   
            !containsCacheStatus(secondCompilationDriverLog, "cache_status_t::found")); `  or `EXPECT_TRUE(firstCompilationDriverLog.empty());`, this test can work.